tox.ini

```diff
@@ -2,17 +2,21 @@
 linting_folders=src/wagtailbakery/ tests/ examples/
 
 [tox]
-envlist=py{36,37,38,39}-django{22,30,31}-wagtail{210,211,master}
+envlist=
+  py{36,37,38}-django{22,30,31}-wagtail211,        # Wagtail 2.11 compatability matrix
+  py{36,37,38,39}-django{22,30,31,32}-wagtail213,  # Wagtail 2.13 compatability matrix
+  py39-django32-wagtailmain                        # Wagtail main latest compatible version
```

Comment:
We normally test for LTS, current and current-1, so right now it would be 2.11, 2.12 and 2.13.

It just so happened that the previous matrix only had Wagtail 2.11 and 2.12 because 2.11 was both LTS and current-1.

That being said, the README also says _Wagtail versions as supported by Wagtail_ and 2.12 isn't officially supported anymore. So I guess we will just leave it like this and update the README later.

Also you can drop `py36` and `py37` for Wagtail 2.11

Reply:
Done! Updated the readme too :+1: